c2corg_ui/static/js/map/map.js

```diff
@@ -59,7 +59,14 @@ app.module.directive('appMap', app.mapDirective);
  * @export
  * @ngInject
  */
-app.MapController = function($scope, mapFeatureCollection) {
+app.MapController = function($scope, mapFeatureCollection, $attrs) {
```

Comment:
Isn't ``$attrs`` missing in the params?
Instead of using it, I think it would be more consistent to add ``zoom`` in the ``scope`` param of the map directive.